projects/plugins/jetpack/modules/search/instant-search/lib/styles/_animations.scss

```diff
@@ -0,0 +1,41 @@
+@keyframes slideInUp {
+	from {
+		transform: translate3d( 0, 100vh, 0 );
+		visibility: visible;
+	}
+
+	to {
+		transform: translate3d( 0, 0, 0 );
+	}
+}
+
+@keyframes slideOutDown {
+	from {
+		transform: translate3d( 0, 0, 0 );
+	}
+
+	to {
+		transform: translate3d( 0, 100vh, 0 );
+		visibility: hidden;
+	}
+}
+
+@keyframes fadeIn {
```

Comment:
Hi! These end up in global scope in CSS, so anyone else (like Gutenberg or Jetpack itself) defining the same keyframe ends up overwriting it. You'll potentially get subtle bugs depending on CSS loading order.

https://github.com/Automattic/jetpack/search?l=SCSS&q=fadeIn

Recommend to prefix with something like `jetpack_search_`

Reply:
Hi @simison, thanks for the heads up! This has now been fixed.